filebeat/prospector/redis/config.go

```diff
@@ -0,0 +1,12 @@
+package redis
+
+var defaultConfig = Config{
+	Username: "",
+	Password: "",
+}
+
+type Config struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Config should have comment or be unexported